packages/api/test/functional/disclaimers.spec.js

```diff
@@ -124,8 +112,11 @@ test('POST /auth/register returns an error when the user does not accept all ter
 	client,
 	assert,
 }) => {
+	const { userJson } = await createUser();
+	userJson.email = 'emailtest@disclaimers.com';
```

Comment:
Esse email é utilizado pra algo nesse teste?
Caso seja,  sugiro a implementação abaixo.
Caso contrário, sugiro a remoção desta segunda linha 🙂 
```suggestion
	const { userJson } = await createUser({ append: { email: 'emailtest@disclaimers.com' } });
```

Reply:
Esplicando o teste

Eu crio um usuário com o Sut só pra pegar um objeto json de usuário.
depois eu mudo o email porque vou usar esse objeto para criar um novo usuário na rota `/auth/register`.